src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,12 @@
+$calender-padding: 10px;
+$date-cell-side: 100px;
+$date-cell-gap: 1px;
+
+$days:
+  "monday" 0,
+  "tuesday" 1,
+  "wednesday" 2,
+  "thursday" 3,
+  "friday" 4,
+  "saturday" 5,
+  "sunday" 6;
```

Comment:
i think you should use short versions of the days 

"Implement `start-day` modifier for the `calendar` with `mon`, `tue`, `wed`, `thu`, `fri`, `sat` and `sun` values"

Reply:
Done